README.md

```diff
@@ -33,14 +33,30 @@ You can run the above example with:
 cargo run --example init
 ```
 
-If you want to remove the colorized output and its dependencies, add the
+If you want to remove the colorized output and the timestamps with its respective dependencies, add the
 the following to your Cargo.toml:
 
 ```
 [dependencies.simple_logger]
 default-features = false
 ```
 
+Or to remove only the colorized output but keep the timestamps:
+
+```
+[dependencies.simple_logger]
+default-features = false
+features = ["ts"]
```

Comment:
```suggestion
features = ["timestamps"]
```
considering the suggestion from @piegamesde, if the maintainer decides it's better the long form, this needs to change as well

Reply:
Yeah, I only commented on the first occurrence for redundancy reasons.